### ISSUE TITLE
problem when comparing face

### ISSUE THREAD

**Issue by cucabr** [chen0040/keras-face#8]
I put one more face (my face) in the image bank of the code and I used "face_net_demo.py" for tests, however, the distance is at **1.0380375385284424** (threshold = 0.7)
What can I do wrong? How can I see the validation loss of keras?

ps: in the grayscale image I'm wearing glasses while in the other I'm not.

verify this
![y7ftQvwV-](https://user-images.githubusercontent.com/1984778/58707716-13103480-838c-11e9-800b-061dd15d0eb5.jpg)
on this
![334280345SP](https://user-images.githubusercontent.com/1984778/58707723-186d7f00-838c-11e9-93e9-a95ba22a829e.jpeg)
